tsconfig.cjs.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "target": "ES2022",                               
+    "module": "CommonJS",                             
+    "declaration": true,                              
+    "outDir": ".",                            
+    "esModuleInterop": true,                           
```

Comment:
We tried to get by without esModuleInterop, is this required?

Reply:
If I try to build it without I get this error:

```
 error TS1259: Module '""' can only be default-imported using the 'allowSyntheticDefaultImports' flag

10 import yargs from 'yargs/yargs'
          ~~~~~
```

Reply:
goddamit! ok than we'll merge it with esModuleInterop set